Source: maxice8
Upstream: no (feel free to)
Reason: fixes test for tab chars

--- a/scripts/opensuse-11.1/post-install.sh
+++ b/scripts/opensuse-11.1/post-install.sh
@@ -42,8 +42,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- a/scripts/opensuse-11.2/post-install.sh
+++ b/scripts/opensuse-11.2/post-install.sh
@@ -60,8 +60,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- a/scripts/opensuse-11.3/post-install.sh
+++ b/scripts/opensuse-11.3/post-install.sh
@@ -39,8 +39,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- a/scripts/opensuse-12.3/post-install.sh
+++ b/scripts/opensuse-12.3/post-install.sh
@@ -38,8 +38,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- a/scripts/opensuse-13.1/post-install.sh
+++ b/scripts/opensuse-13.1/post-install.sh
@@ -38,8 +38,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
--- a/scripts/centos-7/post-install.sh
+++ b/scripts/centos-7/post-install.sh
@@ -39,8 +39,8 @@ install -m 600 /dev/null $prefix/var/log/tallylog
 
 cd $prefix/var/log
 for n in messages secure maillog spooler;do
-	[ -f $n ] && continue
-	umask 066 && touch $n
+    [ -f $n ] && continue
+    umask 066 && touch $n
 done
 cd /
 touch $prefix/var/log/lastlog
--- a/scripts/opensuse-42.2/post-install.sh
+++ b/scripts/opensuse-42.2/post-install.sh
@@ -38,8 +38,8 @@ gpgcheck=1
 EOF
 
 if [ $ARCH = "i386" ] ; then
-	echo "  Setting architecture to i686"
-	sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
+    echo "  Setting architecture to i686"
+    sed -i 's/\(# \)\?arch = .*/arch = i686/' ${prefix}/etc/zypp/zypp.conf
 fi
 
 
